src/js/tabs/osd.js

```
@@ -3281,6 +3472,209 @@ osd.initialize = function (callback) {
+                            if (
+                                testX < 1 ||
+                                testX + elementWidth > OSD.data.displaySize.x - 1 ||
+                                testY < 1 ||
+                                testY > OSD.data.displaySize.y - 2
+                            )
+                                break;
+                            let canPlace = true;
+                            for (let i = 0; i < elementWidth && canPlace; i++) {
+                                const checkPos = testY * OSD.data.displaySize.x + testX + i;
+                                const cell = OSD.data.preview[checkPos];
+                                if (cell && cell[0] && cell[0].index !== fieldChanged.index) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)